pom.xml

```diff
@@ -164,6 +164,12 @@
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-network-store-client</artifactId>
             <version>${network-store-client.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.springframework</groupId>
+                    <artifactId>spring-webmvc</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Why do we need this exclusion ? should it be in the network-store-client project ?

Reply:
We need this exclusion because network-store brings a dependency to webmvc and if you have both webmvc and webflux it creates some bad behavior and requests explode more often than not

Reply:
not needed anymore after https://github.com/powsybl/powsybl-network-store/commit/e3ddb7aef6ce6bdff1a0fd6f9b4213749e2bbde2
(tested by removing the exclusion and running mvn dependency:tree)

before
```
[INFO] +- com.powsybl:powsybl-network-store-client:jar:1.0.0-SNAPSHOT:compile
[INFO] |  +- org.springframework.boot:spring-boot-starter-web:jar:2.2.7.RELEASE:compile
[INFO] |  |  +- (org.springframework.boot:spring-boot-starter:jar:2.2.7.RELEASE:compile - omitted for duplicate)
[INFO] |  |  +- (org.springframework.boot:spring-boot-starter-json:jar:2.2.7.RELEASE:compile - omitted for duplicate)
[INFO] |  |  +- org.springframework.boot:spring-boot-starter-tomcat:jar:2.2.7.RELEASE:compile
[INFO] |  |  |  +- (jakarta.annotation:jakarta.annotation-api:jar:1.3.5:compile - omitted for duplicate)
[INFO] |  |  |  +- org.apache.tomcat.embed:tomcat-embed-core:jar:9.0.34:compile
[INFO] |  |  |  +- org.apache.tomcat.embed:tomcat-embed-el:jar:9.0.34:compile
[INFO] |  |  |  \- org.apache.tomcat.embed:tomcat-embed-websocket:jar:9.0.34:compile
[INFO] |  |  |     \- (org.apache.tomcat.embed:tomcat-embed-core:jar:9.0.34:compile - omitted for duplicate)
[INFO] |  |  +- (org.springframework.boot:spring-boot-starter-validation:jar:2.2.7.RELEASE:compile - omitted for duplicate)
[INFO] |  |  +- (org.springframework:spring-web:jar:5.2.6.RELEASE:compile - omitted for duplicate)
[INFO] |  |  \- org.springframework:spring-webmvc:jar:5.2.6.RELEASE:compile
[INFO] |  |     +- (org.springframework:spring-aop:jar:5.2.6.RELEASE:compile - omitted for duplicate)
[INFO] |  |     +- (org.springframework:spring-beans:jar:5.2.6.RELEASE:compile - omitted for duplicate)
[INFO] |  |     +- (org.springframework:spring-context:jar:5.2.6.RELEASE:compile - version managed from 5.2.2.RELEASE; omitted for duplicate)
[INFO] |  |     +- (org.springframework:spring-core:jar:5.2.6.RELEASE:compile - omitted for duplicate)
[INFO] |  |     +- (org.springframework:spring-expression:jar:5.2.6.RELEASE:compile - omitted for duplicate)
[INFO] |  |     \- (org.springframework:spring-web:jar:5.2.6.RELEASE:compile - omitted for duplicate)
```

after
```
[INFO] +- com.powsybl:powsybl-network-store-client:jar:1.0.0-SNAPSHOT:compile
[INFO] |  +- org.springframework:spring-web:jar:5.2.6.RELEASE:compile
[INFO] |  |  +- (org.springframework:spring-beans:jar:5.2.6.RELEASE:compile - version managed from 5.2.2.RELEASE; omitted for duplicate)
[INFO] |  |  \- (org.springframework:spring-core:jar:5.2.6.RELEASE:compile - omitted for duplicate)
```